javaslang/src/main/java/javaslang/collection/Stream.java

```diff
@@ -980,6 +980,17 @@ default int length() {
     }
 
     @Override
+    default Stream<T> leftPadTo(int length, T element) {
+        if (length <= 1) {
```

Comment:
the `if (length <= 1)` is required as it is the termination condition of recursion.

it has to be `length <= 1` instead of  `length <= 0` because I've already prepend an `element` before the recursion call.

I'll try to make it more explicit.